dashboard/config/scripts/netsim.script

```diff
@@ -3,5 +3,19 @@ description_short ""
 description ""
 hidden 'true'
 
+stage 'Sending Bits'
+level 'blockly:NetSim:unit1_lesson04'
```

Comment:
We're trying to deprecate the levels.js method of defining levels and move towards using Level Builder for all level creation.  I don't know how easy this will be to switch on for netsim, but it seems like it would be easier to start down that path now rather than try to upgrade everything later.